composer.json

```diff
@@ -1,7 +1,8 @@
 {
 	"name": "hm-base",
-	"license": "GPL V3+",
+	"license": "GPL-3.0-or-later",
 	"minimum-stability": "stable",
```

Comment:
So, what about this instead?

```json
    "minimum-stability": "dev",
    "prefer-stable": true,
```

Reply:
What is the benefit of this instead of removing that line and leaving it at `stable` by default?

Reply:
This would allow us to fetch non-stable versions when we need/want that. If there is a stable version according to our constraint, we will always get that. If there is not, then Composer will check for dev versions.

Reply:
Makes a lot of sense, thanks 👍